service/src/integrationTest/kotlin/fi/espoo/evaka/pis/service/MockVtjClientService.kt

```diff
@@ -0,0 +1,99 @@
+// SPDX-FileCopyrightText: 2017-2023 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.pis.service
+
+import fi.espoo.evaka.shared.dev.DevPerson
+import fi.espoo.evaka.vtjclient.service.vtjclient.IVtjClientService
+import fi.espoo.evaka.vtjclient.soap.VTJHenkiloVastaussanoma
+import java.util.concurrent.ConcurrentHashMap
+
+class MockVtjClientService : IVtjClientService {
+
+    companion object {
+        private var queryRequestResponse:
+            ConcurrentHashMap<
+                Pair<String, IVtjClientService.RequestType>, VTJHenkiloVastaussanoma.Henkilo
+            > =
+            ConcurrentHashMap()
+
+        private var queryCounts:
+            ConcurrentHashMap<Pair<String, IVtjClientService.RequestType>, Int> =
+            ConcurrentHashMap()
+
+        fun resetQueryCounts() {
+            this.queryCounts.clear()
+        }
+
+        fun addPERUSSANOMA3Request(person: DevPerson) {
```

Comment:
I would rename`addPERUSSANOMA3Request` and `addHUOLTAJAHUOLLETTAVARequest` into `add...Expectation`